src/transformers/models/auto/configuration_auto.py

```diff
@@ -305,7 +306,7 @@
         ("dialogpt", "DialoGPT"),
         ("distilbert", "DistilBERT"),
         ("dit", "DiT"),
-        ("donut", "Donut"),
+        ("donut-swin", "Donut"),
```

Comment:
This one should be `"DonutSwin"`, you can leave another one for the full model (`("donut", "Donut")`).

Reply:
Are you sure here? Because I see other models like ("fsmt", "FairSeq Machine-Translation") => isn't this the place where we define what appears in the big table of tasks?

Reply:
Ok, so when I add ("donut", "Donut") and ("donut-swin", "Donut") here, the big table gets updated to DonutSwin (which should actually be Donut)

Reply:
Gently pinging @sgugger here, as it's my last question before this PR can be merged.